sample-queries/sample-queries.json

```diff
@@ -1679,6 +1679,65 @@
         "tip": "enable ExternalItem.Read.All",
         "skipTest": false
     },
+    {
+        "id": "26743d2a-2306-40f9-b3e8-3f857af69269",
+        "category": "Compliance (beta)",
+        "method": "GET",
+        "humanName": "List ediscoveryCases",
+        "requestUrl": "/beta/compliance/ediscovery/cases",
+        "docLink": "https://docs.microsoft.com/graph/api/ediscoverycase-list?view=graph-rest-beta",
+        "skipTest": false
+    },
+    {
+        "id": "d3f13cd9-2c89-4876-a88c-081e41370a02",
```

Comment:
Add a tip message to tell the user which permission they need to get the queries running.

Reply:
I presume you're referring to the Graph Permissions like User.ReadAll.  Is that necessary since I don't think we even require that?  All permissioning is handled inside of compliance and Graph is simply a passthrough.  Should I link to something like this?  https://docs.microsoft.com/en-us/microsoft-365/compliance/assign-ediscovery-permissions?view=o365-worldwide.

Reply:
This is a bit of an ongoing conversation with Irvine.  Would be nice if we can publish without any tips about permissions until we get to that.

Reply:
For this, I meant add a tip value letting the user know the permissions required to run the query, but this is not important if we shall populate the Modify Permissions tab with the permissions required. but the tip would be:

`tip: "Go to Modify Permissions tab and consent to User.Read permission to run this query" `

Reply:
I added the tip